### PR TITLE
ValuePicker: refactors story from knobs to control

### PR DESCRIPTION
**Pull request by thisisobate**
**What this PR does / why we need it**:
This PR refactors story from knobs to control.
**Which issue(s) this PR fixes**:

relates to #30182

### CODE REVIEW

File: packages/grafana-ui/src/components/ValuePicker/ValuePicker.story.tsx [hunk 1]
```diff
@@ -15,33 +15,42 @@ export default {
...
 
-export const simple = () => {
-  const label = text('Label', 'Pick an option', VISUAL_GROUP);
-  const variant = select('Variant', variants, 'primary', VISUAL_GROUP);
-  const size = select('Size', sizes, 'md', VISUAL_GROUP);
-  const isFullWidth = boolean('Is full width', false, VISUAL_GROUP);
-  const icon = getIconKnob();
-  const menuPlacement = select('Menu placement', ['auto', 'bottom', 'top'], 'auto', VISUAL_GROUP);
-
+export const Simple: Story<ValuePickerProps<T>> = (args) => {
```
Comment (thisisobate) [inline packages/grafana-ui/src/components/ValuePicker/ValuePicker.story.tsx:42]: `<T>` gives off this error: Cannot find name 'T'. I tried removing T completely but I figured it's not the best approach. Do you have any idea of how we could solve this? @Clarity-89

Comment (Clarity-89) [reply packages/grafana-ui/src/components/ValuePicker/ValuePicker.story.tsx:42]: `T` in this case points to the value of the option object, which is a `string`.

**Review by Clarity-89 — APPROVED** [review]
Looks good!

**Comment by thisisobate** [comment]
oh...ok.
Thanks!

**Pull request merged**